actions/release/create-lang-fam-release-body/action.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Create Language Family Release Notes'
+
+description: |
+  Creates release notes for a language family
+
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```

Comment:
I think you should declare the `release_body` output as it shown in the docs: https://help.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs